platform/web/export/export_plugin.cpp

```diff
@@ -330,6 +330,7 @@ void EditorExportPlatformWeb::get_export_options(List<ExportOption> *r_options)
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_template/release", PROPERTY_HINT_GLOBAL_FILE, "*.zip"), ""));
 
 	r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "variant/extensions_support"), false)); // Export type.
+	r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "variant/compatibility_mode"), false)); // Compatibility (ie. run without COEP/COOP headers).
```

Comment:
```suggestion
	r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "variant/compatibility_mode"), false)); // Compatibility (i.e. run without COEP/COOP headers).
```
But same question as I wrote on the docs. Should we put the focus on headers, or on the actual feature provided by Godot (which browsers impose specific header requirements on)?